spec-main/api-browser-window-spec.ts

```diff
@@ -3267,6 +3280,20 @@ describe('BrowserWindow module', () => {
       await unmaximize;
     });
 
+    it('emits an event when a transparent window is unmaximized', async () => {
+      const w = new BrowserWindow({
+        show: false,
+        frame: false,
+        transparent: true
+      });
+
+      const unmaximize = emittedOnce(w, 'unmaximize');
+      w.show();
+      w.maximize();
+      w.unmaximize();
+      await unmaximize;
```

Comment:
```suggestion
      const maximize = emittedOnce(w, 'maximize');
      const unmaximize = emittedOnce(w, 'unmaximize');
      w.show();
      w.maximize();
      await maximize;
      w.unmaximize();
      await unmaximize;
```

On Ubuntu there's a slight delay when maximizing a window, and `unmaximize` is a no-op if the window isn't maximized, so for this test to pass on Ubuntu it needs to wait for maximize before unmaximizing.